.vib/parse/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - node
+  - parse-server
+  - python
+  - mongosh
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/parse
```

Comment:
I think you missed adding here the `/opt/bitnami/parse/tmp` dir.

Reply:
You're right, I've added it.